composer.json

```diff
@@ -1,20 +1,24 @@
 {
     "name": "laminas/laminas-mvc-plugins",
-    "description": "Collection of all stable laminas-mvc plugins served via separate packages.",
     "type": "metapackage",
-    "license": "BSD-3-Clause",
+    "description": "Collection of all stable laminas-mvc plugins served via separate packages.",
```

Comment:
We do not want another normalization because it was already done during the migration from ZF.